src/com/facebook/buck/apple/ProjectGenerator.java

```diff
@@ -137,6 +139,7 @@
 
   private static final Logger LOG = Logger.get(ProjectGenerator.class);
   private static final String BUILD_WITH_BUCK_TEMPLATE = "build-with-buck.st";
+  private static final String BUILD_WITH_BUCK_FIX_UUID_TEMPLATE = "build-with-buck-fix-uuid.st";
```

Comment:
I think this should be renamed to `fix-uuid.st` as this script does not actually build anything.